OpenUtau/Controls/NotesCanvas.cs

```diff
@@ -98,9 +100,13 @@ class NotesCanvas : Canvas {
         private HashSet<UNote> selectedNotes = new HashSet<UNote>();
         private Geometry pointGeometry;
 
+        private bool showGhostNotes = true;
+        private List<UPart> otherPartsInView = new List<UPart>();
+
         public NotesCanvas() {
             ClipToBounds = true;
             pointGeometry = new EllipseGeometry(new Rect(-2.5, -2.5, 5, 5));
+            showGhostNotes = Convert.ToBoolean(Preferences.Default.ShowGhostNotes);
```

Comment:
This makes preference change only apply after relaunch. Could we move the check to RefreshGhostNotes()?

Reply:
Fixed, and that move does load preferences correctly